x-pack/plugins/fleet/server/services/epm/registry/proxy.ts

```diff
@@ -38,11 +38,15 @@ export function getProxyAgent(options: GetProxyAgentParams): ProxyAgent {
 export function getProxyAgentOptions(options: GetProxyAgentParams): HttpsProxyAgentOptions {
   const endpointParsed = new URL(options.targetUrl);
   const proxyParsed = new URL(options.proxyUrl);
+  const authValue = proxyParsed.password
```

Comment:
I assume having just a username but no password is not valid?

Reply:
I think it is valid; just a bad idea. I should probably switch to testing for username since that's the only required part